### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Local IIS Add Server_Keyboard: After clicking on Add Server button focus is going to hambergur menu instead of content of Add Server

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#65]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)

Keyboard.
MAS ID Violated: MAS 2.4.3

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till Local IIs link  on the top navigation bar and press enter
4.Tab till add Server button and press enter
4.Check whether the focus order is correct or not

Actual Result:
After clicking on Add Server button focus is going to hamburger  menu instead of content of Add Server

Expected Result:
Keyboard focus should move inside the content of Add Server

User Impact:
Keyboard only users find difficulty in using the application if  focus order is incorrect 

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Reference video
[Bug_65_Focus order is incorrect.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317453/Bug_65_Focus.order.is.incorrect.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.4.3; #C+E-H; #.NETWebsites; #Edge

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
